common/djangoapps/third_party_auth/models.py

```diff
@@ -225,7 +227,7 @@ def get_social_auth_uid(self, remote_id):
         return remote_id
 
     @classmethod
-    def get_register_form_data(cls, pipeline_kwargs):
+    def get_register_form_data(cls, pipeline_kwargs, **kwargs):
```

Comment:
I've tried this, but was getting failures in a bunch of tests which started to look awfully awkward to solve, so I tried to work around it with this technique. Going to take another look at this solution and get back to you.